src/SFA.DAS.ApprenticeCommitments/Data/Models/CommitmentStatement.cs

```diff
@@ -35,8 +35,11 @@ private CommitmentStatement()
         public bool? ApprenticeshipDetailsCorrect { get; private set; }
         public bool? HowApprenticeshipDeliveredCorrect { get; private set; }
         public bool ApprenticeshipConfirmed => ConfirmedOn.HasValue;
+        
+        public DateTime ConfirmBefore => CommitmentsApprovedOn.AddDays(14);
```

Comment:
Should 14 be a config value?

Reply:
Yes it does need to be configurable.  I'm going do to that as part of CS-647 (coming up next 😄) as that's when we'll need to persist the date to the database.